ui/src/app/pages/setting/component/adaptertype/adaptertype-list/facility-adapter-list.component.html

```diff
@@ -0,0 +1,551 @@
+                                <clr-dg-cell>{{command.triggerCycle}} minutes</clr-dg-cell>
+                            </clr-dg-row>
+                        </clr-datagrid>
+                    </div>
+                </section>
+            </form>
+        </div>
+    </clr-wizard-page>
+</clr-wizard>
+
+<clr-modal [(clrModalOpen)]="showDetailPageShow" [clrModalSize]="'lg'">
+    <h3 class="modal-title">Adapter config</h3>
+    <div class="modal-body">
+        <form >
+            <section class="form-block">
+                <div class="form-group">
+                    <label>Type:</label>
+                    <span>{{translateTypeName(showDetaileAdapter.type)}}</span>
+                </div>
+
+                <div class="form-group">
+                    <label>Display name:</label>
+                    <span>{{showDetaileAdapter.displayName }}</span>
+                </div>
+                <div class="form-group">
+                    <label>Description:</label>
+                    <textarea clrTextarea [(ngModel)]="showDetaileAdapter.description" name="description" readonly></textarea>
+                </div>
+                <div class="form-group">
+                    <label>Commands:</label>
+                    <clr-datagrid style="width: -webkit-fill-available;">
+                        <clr-dg-column>Name</clr-dg-column>
+                        <clr-dg-column>Description</clr-dg-column>
+                        <clr-dg-column>Trigger Cycle</clr-dg-column>
+                    
+                        <clr-dg-row *ngFor="let command of showDetaileAdapter.commands">
+                            <clr-dg-cell>{{command.command}}</clr-dg-cell>
+                            <clr-dg-cell title="{{command.description}}" style="max-width: 200px;text-overflow:ellipsis;overflow:hidden;white-space: nowrap;">{{command.description}}</clr-dg-cell>
+                            <clr-dg-cell>{{command.triggerCycle}} minutes</clr-dg-cell>
+                        </clr-dg-row>
+                    </clr-datagrid>
+                </div>
+                <div class="form-group">
+                    <label>Topic:</label>
+                    <span>{{showDetaileAdapter.topic }}</span>
+                </div>
+                <div class="form-group">
+                    <label>subCategory:</label>
+                    <span>{{showDetaileAdapter.subCategory }}</span>
+                </div>
+                <div class="form-group">
+                    <label>Queue name:</label>
+                    <span>{{showDetaileAdapter.queueName }}</span>
+                </div>
+            </section>
+        </form>
+    </div>
+</clr-modal>
+
+<clr-modal [(clrModalOpen)]="confirmDeleteShow">
+    <h3 class="modal-title">Operation Tips</h3>
+    <div class="modal-body">
+        <p>Are you sure you want to delete this data? This action is permenant and cannot be recovered.</p>
```

Comment:
Please double check if you need to delete this adapter. This action is permanent and CANNOT be recovered